src/fake_data/provide_hass.ts

```diff
@@ -134,7 +134,8 @@ export const provideHass = (
     },
     panelUrl: "lovelace",
 
-    language: getActiveTranslation(),
+    language: getLocalLanguage(),
+    selectedLanguage: getLocalLanguage(),
```

Comment:
Let's call it once and store it in a variable, then assign `language` and `selectedLanguage` to that var.